tests/conftest.py

```diff
@@ -626,7 +626,8 @@ def html(content="", same_origin=True):
 def iframe():
     """Return a factory for <iframe> with the given src."""
     def iframe(src=""):
-        return f'<iframe src="{src}" />'
+        # Geolocation is required for Geolocation testing in iframes.
+        return f'<iframe allow="geolocation *" src="{src}" />'
```

Comment:
Should we parameterize enabling geolocation vs. enabling it for all? Is this function going to be reused elsewhere or for negative tests?